doc/sphinx/how-to/how-to-specify-startup-apps.md

```diff
@@ -0,0 +1,52 @@
+# How to Specify Startup Applications
+
+**Note**: This builds on top of [Write Your First Wayland
+Compositor](../write-your-first-wayland-compositor.md).
```

Comment:
I think that the purpose of the howto should come first. Then we should state that we assume that the reader has completed the tutorial